about.html

```diff
@@ -0,0 +1,269 @@
+        <p class="text-black-50 mx-5 mt-4 text-center">
+          TNW Creative is here to help you build your brand and navigate the
+          world of tech. Our team of creatives will design your strategy around
+          your needs
+        </p>
+        <p class="only-desktop text-black-50 mx-5 mt-4 text-center">
+          TNW Talks are known for being daring, bold, inspirational, and
+          aspirational. Our 150+ expert speakers bring fresh new insights from
+          the tech industry that can help transform the world of tomorrow.
+        </p>
+        <div class="card m-5 mt-2 border-3 border-white bg-light">
+          <div class="card-body mx-auto">
+            <img
+              src="img/The_Next_Web_Logo.png"
+              alt="The_Next_Web_Logo"
+              class="logo"
+            />
+          </div>
+        </div>
+      </section>
+      <hr />
+      <section id="previous-conference">
+        <p class="mx-4 mt-4 display-6 secondary-color text-center">
+          See the past, TNW conference, NOW.
+        </p>
+        <div class="short-line mx-auto"></div>
+        <p class="mx-4 mb-5 text-black-50 text-center">
+          Take a look at the last two TNW Global Conference which took place
+          always In • Amsterdam, June 16 & 17.
+        </p>
+        <ul class="conferences-list m-4 mb-5">
+          <li>
+            <img
+              src="img/conferences/conference2020.jpg"
+              alt="conferece 2020 image"
+              class="img-thumbnail rounded mx-auto d-block"
+            />
+            <div class="centered">
+              <h5>
+                The Next Web conference <br />
+                <br />
+                2020
+              </h5>
+            </div>
+            <div class="bg-blur"></div>
+          </li>
+          <li>
+            <img
+              src="img/conferences/conference2018.jpg"
+              alt="conferece 2018 image"
+              class="img-thumbnail rounded mx-auto d-block"
+            />
+            <div class="centered">
+              <h5>
+                The Next Web conference <br /><br />
+                2018
+              </h5>
+            </div>
+            <div class="bg-blur"></div>
+          </li>
+        </ul>
+      </section>
+      <section
+        id="partners"
+        class="partners d-flex flex-column justify-content-center align-items-center only-mobile"
+      >
+        <h3 class="text-center mt-3">Partners</h3>
+        <div class="short-line mx-auto mt-3"></div>
+        <div class="partners-logos mt-3">
+          <img src="img/partners/mozilla.svg" alt="mozilla icon" />
+          <img src="img/partners/google.svg" alt="google icon" />
+          <img src="img/partners/naver.svg" alt="naver icon" />
+          <img src="img/partners/tokopedia.png" alt="tokopedia icon" />
+          <img src="img/partners/airbnb.svg" alt="airbnb icon" />
+        </div>
+      </section>
```

Comment:
- [ ] [REQUIRED] Kindly ensure that the About page footer for the mobile is well aligned and properly centered, this will make your design look more professional.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![Screenshot from 2022-04-23 00-33-40](https://user-images.githubusercontent.com/58692590/164817564-ad21127f-8ba9-4494-8d4f-fe2812d6731d.png) | ![Screenshot from 2022-04-23 00-33-27](https://user-images.githubusercontent.com/58692590/164817632-063b1d55-d707-4d26-b869-23d279a21051.png)

Reply:
It's done perfectly now!🔥.